src/modules/Transcript/index.tsx

```diff
@@ -363,7 +380,29 @@ function useEventUpdates(eventId: string) {
     return eventUpdateQuery;
 }
 
-function useEventData(eventId: string, eventUpdateQuery: QueryResult<EventUpdatesQuery, EventUpdatesQueryVariables>) {
+function useLatestEventForTicker(ticker = '') {
+    const latestEventForTickerQuery = useQuery<LatestEventForTickerQuery, LatestEventForTickerQueryVariables>({
+        query: gql`
+            query LatestEventForTicker($filter: LatestEventFilter!) {
+                latestEventForTicker(filter: $filter) {
+                    id
+                }
+            }
+        `,
+        pause: !ticker,
+        requestPolicy: 'cache-first',
+        variables: {
+            filter: {
+                ticker: ticker,
```

Comment:
Can just be `ticker`

Reply:
good catch originally I had change to throw in a default string, but then I updated the args